docs/guide/recovery.md

```diff
@@ -37,11 +37,11 @@ Once you've made sure your device's bootloader is unlocked and have installed a
   - **Magisk/KernelSU Manager**: open the Magisk/KernelSU Manager app and select "Reboot to recovery" option in the Toolbar menu.
 - Open the Install menu, select the UN1CA package you previously downloaded and flash it. UI might vary depending which recovery you've installed.
 - Your specific device installer might have a "Post-install" feature enabled which will automatically reboot your device a few times. Don't worry about it.
-- Once the install has finished, open the Wipe → Format Data menu, type "yes" and confirm. **This step is only required the very first time you install the ROM.**
+- Once the installation has finished, open the Wipe → Format Data menu, type "yes" and confirm. **This step is only required the very first time you install the ROM.**
 
-If you have followed every step correctly, the ROM has been installed succesfully and you can now exit recovery by selecting Reboot → System.
+If you followed every step correctly, the ROM has been installed succesfully and you can now exit recovery by selecting Reboot → System.
 
-If any other error/issue occurs, please refer to the [Troubleshooting]({% link faq/index.md %}) section.
+If you encounter any error/issue, please refer to the [Troubleshooting]({% link faq/index.md %}) section.
```

Comment:
This must match the Download mode guide:
https://github.com/BlackMesa123/UN1CA/blob/073aed5b12625bd6e5550f4f75a107fd19423fe2/docs/guide/download.md?plain=1#L65